examples/miral-shell/shell_main.cpp

```diff
@@ -172,6 +175,39 @@ int main(int argc, char const* argv[])
         return false;
     };
 
+    auto locate_pointer = miral::LocatePointer{false}
+                              .on_locate_pointer([](auto, auto) { mir::log_info("Locate pointer!"); })
```

Comment:
So the only thing this example does is spam the log with a fixed string? Is that a good example?

If writing to the console is the best we can do we should, at  least, be outputting some location information.